libcnb-test/README.md

```diff
@@ -46,3 +46,27 @@ fn call_test_fixture_service(addr: std::net::SocketAddr, payload: &str) -> Resul
    unimplemented!()
 }
 ```
+
+## Configuration
```

Comment:
It seems configuring the target via `Cargo.toml` works for both `libcnb-test` and `libcnb-cargo` CLI usage, however the config is only documented for `libcnb-test`. Should this be added to the `libcnb-cargo` README too? Somewhere else instead and linked to from both?

(Perhaps another reason why skipping allowing this configuration might be simpler for now, unless we really need it?)

Reply:
I moved the main docs to `libcnb-cargo` and added a reference to them in `libcnb-test`'s README. I don't think we need to duplicate the docs, `libcnb-test` is only really useful if you use `libcnb-cargo` for your release buildpack too.

Reply:
Obsolete after dfc90e8.